app/styleguide/routes/attributegrid.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div :class="b('spacing')">
+    <h1>Attributegrid</h1>
+    <c-attribute-grid :attributes="[{'name': 'Test', 'content': 'Testvalue'}, {'name': 'Test', 'content': 'Testvalue'}, {'name': 'Test', 'content': 'Testvalue'}, {'name': 'Test', 'content': 'Testvalue'}, {'name': 'Test', 'content': 'Testvalue', url: 'http://www.google.com'}]" />
```

Comment:
- [ ] Please also add to http://localhost:8080/styleguide/product-detail page (`specs`)